concourse/pipelines/windows-image-build-mbr.jsonnet

```diff
@@ -204,11 +232,24 @@ local imgpublishjob = {
         source_gcs_path: job.gcs,
         source_version: 'v((.:source-version))',
         publish_version: '((.:publish-version))',
-        wf: job.workflow,
-        environment: if job.env == 'testing' then 'test' else job.env,
+          wf: job.workflow,
+          environment: if job.env == 'testing' then 'test' else job.env,
       },
     },
-  ],
+  if job.runtests then
+    [
```

Comment:
Fly won't validate these subarrays and jsonnet won't expand them back out into the main one, you have to stitch multiple arrays together to make the tooling happy.